docs/tutorials/assets-tokens/withdrawing-tokens.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Withdraw tokens
```

Comment:
```suggestion
title: Withdraw staked tokens
```

Reply:
Tokens makes me think of ERC20 in general and this is only Vega. Suggest staked tokens as vega tokens could also be vested in Vega or in a general account.